src/diffusers/pipelines/stable_diffusion/pipeline_flax_stable_diffusion_img2img.py

```diff
@@ -164,12 +166,11 @@ def _run_safety_checker(self, images, safety_model_params, jit=False):
 
         return images, has_nsfw_concepts
 
-    def get_timestep_start(self, num_inference_steps, strength, scheduler_state):
+    def get_timestep_start(self, num_inference_steps, strength):
         # get the original timestep using init_timestep
-        offset = self.scheduler.config.get("steps_offset", 0)
```

Comment:
why is `offset` removed? We need to handle this for `DDIMScheduler`

Reply:
The offset cancels itself out. It was also removed from the other img2img pipelines in #1653

Reply:
Oh I see, didn't notice that, thanks!